NEWS.md

```diff
@@ -5,7 +5,9 @@ Julia v1.8 Release Notes
 New language features
 ---------------------
 
-* `Module(:name, false, false)` can be used to create a `module` that does not import `Core`. ([#40110])
+* `(; a, b) = x` can now be used to destructure properties `a` and `b` of `x`. This syntax is equivalent to `a = getproperty(x, :a)`
+  and similarly for `b`. ([#39285])
```

Comment:
This was in my old branch, but not in master, but I think it should still be in the master branch, perhaps it was removed accidentally?

Reply:
The feature is added for v1.7, and this is a release note for v1.8.

Reply:
Oh, so it is. Well, perhaps I should merge this with a different branch or against a different tag or something?

Maybe 7ca1a9047aaaa6a9a31e5025f4728fc1d86a5cf8 ? (one commit before Kristoffer Carlsson updated VERSION)?

Reply:
Please rebase on master. We first merge things against master and later one backport if needed.